test/parallel/test-net-connect-local-error.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+var common = require('../common');
+var assert = require('assert');
+var net = require('net');
+
+var client = net.connect({
+  port: common.PORT + 1,
```

Comment:
I'm not sure which port to use here, it simply should produce an ECONNREFUSED error when connecting.